pkg/values/k8s.go

```diff
@@ -7,27 +7,31 @@ import (
 )
 
 var K8SAPIVersionMap = map[string]string{
-	"1.28": "registry.k8s.io/kube-apiserver:v1.28.2",
-	"1.27": "registry.k8s.io/kube-apiserver:v1.27.6",
-	"1.26": "registry.k8s.io/kube-apiserver:v1.26.9",
-	"1.25": "registry.k8s.io/kube-apiserver:v1.25.14",
+	"1.29": "registry.k8s.io/kube-apiserver:v1.29.0",
+	"1.28": "registry.k8s.io/kube-apiserver:v1.28.4",
+	"1.27": "registry.k8s.io/kube-apiserver:v1.27.8",
+	"1.26": "registry.k8s.io/kube-apiserver:v1.26.11",
+	"1.25": "registry.k8s.io/kube-apiserver:v1.25.16",
 }
 
 var K8SControllerVersionMap = map[string]string{
-	"1.28": "registry.k8s.io/kube-controller-manager:v1.28.2",
-	"1.27": "registry.k8s.io/kube-controller-manager:v1.27.6",
-	"1.26": "registry.k8s.io/kube-controller-manager:v1.26.9",
-	"1.25": "registry.k8s.io/kube-controller-manager:v1.25.14",
+	"1.29": "registry.k8s.io/kube-controller-manager:v1.29.0",
+	"1.28": "registry.k8s.io/kube-controller-manager:v1.28.4",
+	"1.27": "registry.k8s.io/kube-controller-manager:v1.27.8",
+	"1.26": "registry.k8s.io/kube-controller-manager:v1.26.11",
+	"1.25": "registry.k8s.io/kube-controller-manager:v1.25.16",
 }
 
 var K8SSchedulerVersionMap = map[string]string{
-	"1.28": "registry.k8s.io/kube-scheduler:v1.28.2",
-	"1.27": "registry.k8s.io/kube-scheduler:v1.27.6",
-	"1.26": "registry.k8s.io/kube-scheduler:v1.26.9",
-	"1.25": "registry.k8s.io/kube-scheduler:v1.25.14",
+	"1.29": "registry.k8s.io/kube-scheduler:v1.29.0",
+	"1.28": "registry.k8s.io/kube-scheduler:v1.28.4",
+	"1.27": "registry.k8s.io/kube-scheduler:v1.27.8",
+	"1.26": "registry.k8s.io/kube-scheduler:v1.26.11",
+	"1.25": "registry.k8s.io/kube-scheduler:v1.25.16",
 }
 
 var K8SEtcdVersionMap = map[string]string{
+	"1.29": "registry.k8s.io/etcd:3.5.10-0",
 	"1.28": "registry.k8s.io/etcd:3.5.9-0",
 	"1.27": "registry.k8s.io/etcd:3.5.7-0",
 	"1.26": "registry.k8s.io/etcd:3.5.6-0",
```

Comment:
we also add the new version below where we still check for 28

Reply:
for the kind [releases page](https://github.com/kubernetes-sigs/kind/releases/tag/v0.20.0)

> Additional images built for this release:
    1.28: kindest/node:v1.28.0@sha256:b7a4cad12c197af3ba43202d3efe03246b3f0793f162afb40a33c923952d5b31
    

>   NOTE: You must use the @sha256 digest to guarantee an image built for this release, until such a time as we switch to a different tagging scheme. Even then we will highly encourage digest pinning for security and reproducibility reasons.

shall I give it a try without the sha once?